pkg/commands/app.go

```diff
@@ -667,6 +667,19 @@ func NewPluginCommand() *cli.Command {
 				ArgsUsage: "PLUGIN_NAME",
 				Action:    plugin.Uninstall,
 			},
+			{
+				Name:    "list",
+				Aliases: []string{"l"},
+				Usage:   "list installed plugin",
+				Action:  plugin.List,
+			},
+			{
+				Name:      "info",
+				Aliases:   []string{},
```

Comment:
If we don't have any aliases, we can remove this line.
```suggestion
```

Or,

```suggestion
				Aliases:   []string{"i"},
```